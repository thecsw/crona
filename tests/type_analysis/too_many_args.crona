f : bool(a : int) {}

main : void() {
    a : int;
    a = 1 + f(1, 2);
}