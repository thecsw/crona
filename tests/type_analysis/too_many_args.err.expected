FATAL [5,13]: Function call with wrong number of args
FATAL [5,13]: Arithmetic operator applied to invalid operand
Type Analysis Failed
